### PR TITLE
Commenting system logic

### PR DESCRIPTION
**Pull request by MariemBaccari**

### CODE REVIEW

File: app/src/main/java/com/github/sdp/mediato/data/ReviewInteractionDatabase.java [hunk 1]
```diff
@@ -17,6 +20,20 @@ public class ReviewInteractionDatabase {
 
     public static FirebaseDatabase database = FirebaseDatabase.getInstance();
 
+    /**
+     * Adds a comment to a review.
+     * @param tarUsername the username of the user who made the review
+     * @param comment the comment
+     */
+    public static void commentReview(String tarUsername, Comment comment) {
```
Comment (simone-kalbermatter) [inline app/src/main/java/com/github/sdp/mediato/data/ReviewInteractionDatabase.java:28]: What does `tarUsername` mean?

**Review by simone-kalbermatter — CHANGES_REQUESTED** [review]
LGTM, clean, well documented and tested. Just one question.

File: app/src/main/java/com/github/sdp/mediato/data/ReviewInteractionDatabase.java (hunk 1 above)
Comment (MariemBaccari) [reply app/src/main/java/com/github/sdp/mediato/data/ReviewInteractionDatabase.java:28]: Target username. As specified in the java doc, it's the username of the user that made the review which is being commented (as opposed to refUsername - reference username which is the user that is currently logged in). The terminology is used in other database methods.

**Review by simone-kalbermatter — APPROVED** [review]
LGTM!

**Pull request merged**